config_example.sh

```diff
@@ -131,7 +131,10 @@ set -x
 #export PROVISIONING_NETWORK=fd00:1101::0/64
 
 # Provisioning network profile, can be set to "Managed" or "Disabled"
-#PROVISIONING_NETWORK_PROFILE=Managed
+#export PROVISIONING_NETWORK_PROFILE=Managed
+
+# Configure sushy-tools to allow booting from different devices
+#export SUSHY_IGNORE_BOOT_DEVICE=False
```

Comment:
Sushy is a codename that not everyone knows. Let's rename the option and rephrase the comment in terms of a redfish emulator.

Reply:
agreed, have another look